data/jsonschema.md

```diff
@@ -124,13 +124,16 @@ The same logic would apply to the `description` if any where there.
 ## Enum
 
 - **For values**, apply the following rules to ease the translation work:
-  - Use URIs if available.
-  - Avoid colons `:`.
-  - Use prefixes if more than 8-10 values (to judge according to each use
+  - Use controlled vocabulary if possible. Example: [RDA values](http://www.rdaregistry.info/termList/)
+    - Use a prefix to identify the vocabulary, and separate it from the value with underscore `_`
+    - Example for RDA mode of issuance: `rdami_1001`
+  - If no controlled vocabulary exists, use prefixes in case of more than 8-10 values (to judge according to each use
     case).
+  - In general, don't use URIs or values with colons `:`
 - **Form options**:
     - Use the `value`/`label` method **only** if the codes need to be
       translated.
     - In the `label` value, use the same value as the one of the `value` key.
       This is to avoid translating the label and the value, as it is sometimes
       needed.
+    - In case of a `oneOf` (ex: main type/subtype), use the same text for `title` of the 1st-level element, as for `value` and `label`. This will ease the translations.
```

Comment:
```suggestion
    - In case of a list and sublist with `oneOf` (ex: main type/subtype), use the same text for `title` of the 1st-level element, as for `value` and `label`. This will ease the translations.
```